compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEUnrollJamAIEVecMatmul.cpp

```diff
@@ -0,0 +1,448 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <optional>
+
+#include "aievec/AIEVecOps.h"
+#include "iree-amd-aie/IR/AMDAIEDialect.h"
+#include "iree-amd-aie/Target/XCLBinGen.h"
+#include "iree-amd-aie/Transforms/Passes.h"
+#include "iree-amd-aie/Transforms/Utils/AMDAIEUtils.h"
+#include "iree-amd-aie/aie_runtime/AMDAIEEnums.h"
+#include "llvm/Support/raw_ostream.h"
+#include "mlir/Dialect/SCF/IR/SCF.h"
+#include "mlir/Dialect/SCF/Utils/Utils.h"
+#include "mlir/IR/PatternMatch.h"
+#include "mlir/IR/TypeUtilities.h"
+#include "mlir/Pass/PassManager.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+#define DEBUG_TYPE "iree-amdaie-unroll-jam-aievec-matmul"
+
+namespace mlir::iree_compiler::AMDAIE {
+
+namespace {
+
+/// Get the loop count of \p forOp if its loop bounds and step are constant.
```

Comment:
This `\p` notation doesn't seem to be used yet in the codebase and neither in `IREE`, can we just use `` for consistency within the codebase?

Reply:
+1 It's not very readable.

Reply:
Yeah I guess so, especially as we're not generating any doxygen comments. Will update. 

> +1 It's not very readable.

Looks quite nice with the right vim plugin :)